apps/backend/src/db/utils.ts

```diff
@@ -9,6 +9,16 @@ export async function insertDocument(
   });
 }
 
+export async function insertDocuments(
+  documents: Record<string, any>[]
+): Promise<void> {
+  await qldbDriver.executeLambda(async (txn) => {
+    for (const document of documents) {
+      await txn.execute(`INSERT INTO ${tableName} ?`, document);
```

Comment:
Instead of running an insert query for each document, can we do one insert statement that inserts all of them. I'm worried that if they try and upload a lot of documents this might break or be really slow.

You just have to format the string to have a question mark for each document. So I think it would look something like
```
await txn.execute(`INSERT INTO ${tableName} ?, ?, ?`, ...document);
```